From: =?utf-8?q?F=C3=A9lix_Sipma?= <felix+debian@gueux.org>
Date: Wed, 22 Nov 2017 00:55:05 +0100
Subject: use todoman console_script

---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 042d64e..99b6701 100644
--- a/setup.py
+++ b/setup.py
@@ -13,8 +13,8 @@ setup(
     include_package_data=True,
     entry_points={
         'console_scripts': [
-            'todo = todoman.cli:cli',
-        ],
+            'todoman = todoman.cli:cli',
+        ]
     },
     install_requires=[
         'atomicwrites',
